### DISCUSSION TITLE
Add an option to add a todolist

### DISCUSSION THREAD

**Discussion by Oshimada** [dailydotdev/daily#1321] [Feature Requests]
Hello,

I would like to suggest a feature to add a to-do list option alongside or in place of the current shortcut tab. It would be great if it could be synced with existing applications like Trello or Todoist. I am willing to help with the development of this feature if necessary. I look forward to your response.

Have a nice day,
Adnane

**Comment by idoshamun** [comment]
Hi @Oshimada,

Thanks for the suggestion, we will consider it :)
